packages/micromanage/test/entity.test.js

```diff
@@ -0,0 +1,44 @@
+import { Entity } from 'micromanage'
+
+it('must define a schema', () => {
+  class NoSQL extends Entity {}
+
+  expect(() => new NoSQL()).toThrow(/Entity "NoSQL" needs a schema./)
```

Comment:
Okay okay, I know I know, Mongo has schema validations. There's probably quite a bit we can learn from them:
https://docs.mongodb.com/manual/core/schema-validation/